#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_climits.dpatch by Cyril Brulebois <kibi@debian.org>
##
## DP: Fix FTBFS due to missing declaration of PATH_MAX.

@DPATCH@
--- a/extra/alsaplayer.desktop	2008-06-16 20:19:29.000000000 +0100
+++ b/extra/alsaplayer.desktop	2009-08-30 14:54:58.000000000 +0100
@@ -7,6 +7,6 @@
 Exec=alsaplayer
 TryExec=alsaplayer
 Icon=alsaplayer
-MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;application/flac;
+MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;audio/flac;
 Categories=AudioVideo;Audio;Player;
 StartupNotify=true
